sekoia-io-xdr/list_alerts.py

```diff
@@ -8,13 +8,19 @@
 
 def list_alerts(config, params):
     url: str = OPERATION_CENTER_BASE_URL
+
+    if params.get("start_date") and params.get("end_date"):
+        created_at = f"{params['start_date']}, {params['end_date']}"
```

Comment:
With my previous comment, I suggest this modification to avoid `None` string in the `created_at` parameter:
```suggestion
        created_at = f"{params['start_date'] or ''},{params['end_date'] or ''}"
```